shippy-pres-example/shippy-app/js/shippy-storage.js

```diff
@@ -119,19 +132,84 @@ Shippy.Storage = (function() {
 			return styleSheet.href;
 		};
 		let getKey = function (styleSheet) {
```

Comment:
Hmm, I don't get this. Are you defining the function `getKey` twice? This should be invalid actually.